Jenkinsfile_CNP

```diff
@@ -11,11 +11,20 @@ slackChannel = "#ccd-master-builds"
 onMaster {
     try {
         sharedInfrastructurePipeline('ccd', environment.nonProdName, 'nonprod')
-        sharedInfrastructurePipeline('ccd', environment.demoName, 'nonprod')
         sharedInfrastructurePipeline('ccd', environment.prodName, 'prod')
         notifyBuildFixed channel: slackChannel
     } catch (err) {
         notifyBuildFailure channel: slackChannel
         throw err
     }
 }
+
+onDemo {
```

Comment:
Do we want to keep this demo deploy on its own?  Maybe for this initial PR but we should remove in final